fdbserver/VersionedBTree.actor.cpp

```diff
@@ -5239,6 +5239,17 @@ class VersionedBTree {
 			self->m_lazyClearQueue.recover(self->m_pager, self->m_header.lazyDeleteQueue, "LazyClearQueueRecovered");
 			debug_printf("BTree recovered.\n");
 
+			if (ArenaPage::isEncodingTypeEncrypted(self->m_header.encodingType) &&
+			    self->m_encodingType == EncodingType::XXHash64) {
+				// On restart the encryption config of the cluster could be unknown. In that case if we find the Redwood
+				// instance is encrypted, we should use the same encryption encoding.
+				self->m_encodingType = self->m_header.encodingType;
+				self->m_enforceEncodingType = true;
+				TraceEvent(SevWarn, "RedwoodBTreeNodeForceEncryption")
```

Comment:
curious: if this is an expected codepath, why is `warning` level used?

Reply:
good point. will update.